### ISSUE TITLE
No Players showing in Players screen

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#111]
```
What steps will reproduce the problem?

1. Start Squeezer in android 2.2.1
2. Press Players tab - nothing appears .. used to work until i updated to new version
1.2.3

What version of the product are you using? On what operating system?

WIN7 version 1.2.3, Squeeze Player also installed on android device.

Please provide any additional information below.

It's connecting fine to LMS .. I can see all my music. If I use another device on my
LMS network I can control the android device that Squeezer is on and make it playback
the music, but I just can't do it directly on the device for some reason.

Argg .. back to the old Logitech app in the meantime. Squeezer is so much faster :-(
```

Original issue reported on code.google.com by `andrew.w.pollock` on 2014-12-09 01:22:39

**Comment by kaaholst** [comment]
```
Thanks for the report. Will be fixed in 1.2.4, which I shall release later today.

If interested, the fix is https://github.com/nikclayton/android-squeezer/commit/039908df58fa92247eb901df3848e077709ebda9.
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2014-12-09 19:47:38

**Comment by kaaholst** [comment]
```
Excellent ! Thanks for such a fast response :-)
```

Original issue reported on code.google.com by `andrew.w.pollock` on 2014-12-09 20:49:11
